src/commands/build/BaseBuilder.js

```diff
@@ -170,6 +174,38 @@ ${buildStatus.id}
     }
   }
 
+  async wait(buildId, { timeout = 1200, interval = 60 } = {}) {
+    let time = new Date().getTime();
+    log(`Waiting for build ${buildId} to complete, checking status in ${interval}s ...`);
+    await sleep(secondsToMilliseconds(interval));
+    const endTime = time + secondsToMilliseconds(timeout);
+    while (time <= endTime) {
+      const res = await Project.buildAsync(this.projectDir, { current: false, mode: 'status' });
+      const job = fp.compose(
+        fp.head,
+        fp.filter(job => job.id === buildId),
+        fp.getOr([], 'jobs')
+      )(res);
+      switch (job.status) {
+      case 'finished':
+        return job;
+      case 'pending':
+      case 'started':
+      case 'in-progress':
+        log(`Standalone app is still building, checking again in ${interval}s ...`);
```

Comment:
This sometimes gets repeated quite a few times. Maybe we can just show a spinner and some message like this?
```
Waiting for build to complete. You can press Ctrl+C to exit.
```
We use the `@expo/simple-spinner` library for spinners, here's an example of how to use it: https://github.com/expo/exp/blob/95f81c99a163149da9c039148a50777a7a89614e/src/commands/diagnostics.js#L16-L20